### DISCUSSION TITLE
Are the two actions same?

### DISCUSSION THREAD

**Discussion by DFVSQY** [gitkraken/vscode-gitlens#2168] [Q&A] [answered]
### GitLens Version

v12.1.2

### VS Code Version

Version: 1.70.2 (system setup)
Commit: e4503b30fc78200f846c62cf8091b76ff5547662
Date: 2022-08-16T05:35:13.448Z
Electron: 18.3.5
Chromium: 100.0.4896.160
Node.js: 16.13.2
V8: 10.0.139.17-electron.0
OS: Windows_NT x64 10.0.17763

### Git Version

git version 2.37.2.windows.2

### Description

![image](https://user-images.githubusercontent.com/8075421/186447154-7e00ca36-182e-41b3-adfc-98d6c8b9a9cd.png)

I like Gitlens, and i don't know what is difference between the actions? 

Thank you!

### Logs, Screenshots, Screen Captures, etc

![image](https://user-images.githubusercontent.com/8075421/186447642-94ffc3b8-86d6-4567-8fa4-a49354c5cdbb.png)

**Comment by eamodio** [comment, marked as answer]
Yes, those actions should be the same.
